modules/types/src/store.ts

```diff
@@ -59,13 +63,38 @@ export interface FileStorageOptions {
  * have the same behaviour if the `allowDelete` flag is passed; otherwise, any null values or
  * subvalues throws an error.
  */
-export interface IStoreService {
+export interface IStoreServiceOld {
   get(path: string): Promise<any>;
   set(pairs: { path: string; value: any }[], allowDelete?: Boolean): Promise<void>;
   reset?(): Promise<void>;
 }
 
-export interface Store extends IStoreService {
+export interface IStoreService {
+  getAllChannels(): Promise<StateChannelJSON[]>;
+  getStateChannel(multisigAddress: string): Promise<StateChannelJSON | undefined>;
+  getStateChannelByOwners(owners: string[]): Promise<StateChannelJSON | undefined>;
+  getStateChannelByAppInstanceId(appInstanceId: string): Promise<StateChannelJSON | undefined>;
+  saveStateChannel(stateChannel: StateChannelJSON): Promise<void>;
+  getAppInstance(appInstanceId: string): Promise<AppInstanceJson | undefined>;
+  saveAppInstance(multisigAddress: string, appInstance: AppInstanceJson): Promise<void>;
+  // getAppProposals(multisigAddress: string): Promise<AppInstanceJson[]>;
+  // saveAppProposal(appProposal: AppInstanceJson): Promise<void>;
+  getCommitment(commitmentHash: string): Promise<CFCoreTypes.MinimalTransaction | undefined>;
+  saveCommitment(commitmentHash: string, commitment: any[]): Promise<void>;
+  getWithdrawalCommitment(
+    multisigAddress: string,
+  ): Promise<CFCoreTypes.MinimalTransaction | undefined>;
+  saveWithdrawalCommitment(
+    multisigAddress: string,
+    commitment: CFCoreTypes.MinimalTransaction,
+  ): Promise<void>;
+  getExtendedPrvKey(): Promise<string>;
+  saveExtendedPrvKey(extendedPrvKey: string): Promise<void>;
```

Comment:
Are these being used? I'd prefer we remove these methods, it's more secure to keep private keys in memory & not ever write them to disk